MrFish.lua

```diff
@@ -1,22 +1,26 @@
 local __FILE__=tostring(debugstack(1,2,0):match("(.*):1:")) -- Always check line number in regexp and file
 local me,ns=...
---@debug@
+--[===[@debug@
+
```

Comment:
Please, when you commit, restore the debug tags to their original state or they get lost